bigbluebutton-html5/imports/ui/components/utils/hooks/index.js

```diff
@@ -0,0 +1,13 @@
+import { useEffect, useRef } from 'react';
+
+export const usePrevious = (value) => {
```

Comment:
Why did you name this function `usePrevious`? When I read the code above I don't get the relationship between the function name and what it does.

Reply:
Agreed!

What do you suggest? Maybe `usePreviousValue`, `usePreviousState`?